data/tiddlers.json

```diff
@@ -5019,7 +5019,7 @@
         "id_type": "faction_id",
         "modified": "20150303212649830",
         "tags": "factions",
-        "title": "Rebelles & Criminels"
+        "title": "Rebelles \u0026 Criminels"
```

Comment:
Please tell me whether this line generates a bug in your import, because the `\uxxxx` notation is actually in the JSON specification (as well as [this line](#diff-c2bdb59d4fc220abb65f73e8e49d5ab2R4294) ) and generated by the command when adding the new IDs